datasets/us.md

```diff
@@ -0,0 +1,143 @@
+---
```

Comment:
nit: maybe rename the filename (and url) united_states.md or usa.md

Reply:
done